shared/aux.go

```diff
@@ -352,3 +357,27 @@ func fileExists(files []os.DirEntry, workload string) bool {
 
 	return false
 }
+
+// VerifyFileMatchWithPath verify actual file list matches the expected file list
+func VerifyFileMatchWithPath(actualFileList []string, expectedFileList []string, path string) {
```

Comment:
func VerifyFileMatchWithPath(actualFileList , expectedFileList []string, path string) {

Reply:
no need to repeat []string

Reply:
fixed